### ISSUE TITLE
Referring to documentation from within the console

### ISSUE THREAD

**Issue by kschiffer** [TheThingsNetwork/lorawan-stack#2254]
#### Summary
We need to be able to refer to our documentation from within the console.

#### Why do we need this?
Provide help right where it is needed.

#### What is already there? What do you see now?
Documentation

#### What is missing? What do you want to see?
A safe way to refer to the documentation (linking) from within the console.

#### How do you propose to implement this?
- Extend the console configuration to provide a base path to the documentation
- Implement a `<DocLink />` component that will concatenate the base path with the documentation path

#### Can you do this yourself and submit a Pull Request?
@pgalic96

**Comment by kschiffer** [comment]
I envision a usage like this
```jsx
<DocLink ref="/concepts/webhook-templates">
  <Message content={sharedMessages.webhookTemplattes} />
</DocLink>
```

But we should also provide utilities to generate the documentation link outside of components:
```js
const deviceAddrDocLink = docLink('/concepts/webhook-templates')
```

I think the usage of those will mostly revolve around form titles and descriptions, so it's good to think about how we can easily add documentation links in there.

**Comment by pgalic96** [comment]
closed via #2397
